Add documentation for v10.10 license seat count enforcement (#8127)
mattermost/docs: source/manage/admin/installing-license-key.rst

    
    To find the total number of users, go to **System Console > Reporting > Site Statistics**. The total number of users is displayed in the **Total Activated Users** field. The license will be rejected if this value is greater than the value allowed by your license key.

License seat count enforcement
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> From Mattermost server v10.10 onward, system administrators can configure active enforcement of licensed seat count limits using the :ref:`IsSeatCountEnforced <configure/environment-configuration-settings:enforce license seat count>` setting. When this feature is enabled:

Author: ```suggestion
From Mattermost v10.10, system admins can enable active enforcement of licensed seat count limits using the :ref:`IsSeatCountEnforced <configure/environment-configuration-settings:enforce license seat count>` configuration setting. When enabled:
```

    
    To find the total number of users, go to **System Console > Reporting > Site Statistics**. The total number of users is displayed in the **Total Activated Users** field. The license will be rejected if this value is greater than the value allowed by your license key.

License seat count enforcement
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> .. versionadded:: 10.10

Author: ```suggestion
```
